articles/machine-learning-and-deep-learning/index.md

```diff
@@ -0,0 +1,55 @@
+### Introduction
+
+Technology is growing at a fast pace in recent years. Due to this, it is now easy to get lost in the rumpus of technological terms that keep popping up every day. Machine learning and deep learning are among the fast-growing technologies. Besides this two, [Internet of Things (IoT](https://internetofthingsagenda.techtarget.com/definition/Internet-of-Things-IoT)), [blockchain](https://en.wikipedia.org/wiki/Blockchain), and [Artificial Intelligence](https://www.investopedia.com/terms/a/artificial-intelligence-ai.asp) (AI) are among the fast-growing technologies.
+
+New ideas in the tech world have led to various innovations that have sparked the nature of things. Machine learning and deep learning have attracted great attention over the years. The popularity of these two has led to various applications implementing its tools.
+
+This article will focus on understanding what machine learning and deep learning are. Both applications, differences, and functionality will be emphasized in this article.
+
+### Machine Learning and Artificial Intelligence
+
+Machine learning is a technological tool that uses algorithmic interpretations. This has helped in obtaining and using data to make and interpret final decisions. Machine learning processes are also dependent on the subsets of artificial intelligence. Several [synthetic intelligence](https://www.wikiwand.com/en/Synthetic_intelligence) aspects and modern [virtual assistants](https://www.investopedia.com/terms/v/virtual-assistant.asp) are created to achieve this.
+
+Machine learning uses Artificial Intelligence (AI) techniques that assist in decision making. Conventional algorithms in AI aid in performing mathematical computation and predictions. Machine learning depends on fed structured data. Conventional algorithms aids in achieving data mining and predictive analytics in a search.
+
+Unlike machine learning, AI can use [unstructured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html) and still be able to function. Deep learning does mimic the human brain's workings. This is because it uses this data to perform the processing of data. Machine learning aids in the detection of objects, language translation, and decision making. Machine learning is also able to recognize speech and make prompt decisions. The AI makes this possible this allows in no aspect of human supervision.
+
+### Deep Learning
+
+Deep learning is a subset of machine learning. Deep learning uses networks that operate unsupervised and one that uses [structured data](https://www.datamation.com/big-data/structured-vs-unstructured-data.html). This is like the human brain structure as it operates as a subset of machine learning.
+
+This is to means that the algorithms used are complex and are from neural networks. This resemblance is like the layers with neural connections in the human brain.
+
+### How deep learning works
```

Comment:
The feature extraction process of layers in deep learning is a black box. Taking an example where the features have been visualized will help the reader understand better. Consider the following link https://cs231n.github.io/understanding-cnn/. It gives an in-depth understanding of how data is interpreted. Since this is an introductory article, explaining about higher-dimensional vector spaces would not be suitable.